src/App.jsx

```diff
@@ -1,19 +1,17 @@
 // React Imports
 import React from 'react';
-import { HashRouter as Router } from 'react-router-dom';
-// Inrupt Library Imports
-import { SessionProvider } from '@inrupt/solid-ui-react';
+import { BrowserRouter } from 'react-router-dom';
+import { SessionProvider } from '@contexts';
```

Comment:
should move to under `// Context Imports` line, and probably adjust the other context import to be `from @contexts` as well.